<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrajectoryForm</class>
 <widget class="QWidget" name="TrajectoryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>弹道名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>开始位置</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>经度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>纬度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>高度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStartLat">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStartLon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStartAlt">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>结束位置</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>经度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>纬度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>高度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leEndLat">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leEndLon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leEndAlt">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>开始时间</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>时间步长</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsbStep">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="psbCance">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="psbCreate">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>弹道高度</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsbHeight">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1200.000000000000000</double>
   </property>
   <property name="value">
    <double>125.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
